src/Magnum/Audio/CMakeLists.txt

```diff
@@ -56,6 +66,10 @@ endif()
 
 target_link_libraries(MagnumAudio Magnum ${CORRADE_PLUGINMANAGER_LIBRARIES} ${OPENAL_LIBRARY})
 
+if(WITH_SCENEGRAPH)
+    target_link_libraries(MagnumAudio ${MAGNUM_SCENEGRAPH_LIBRARIES})
```

Comment:
Just a wild guess -- the `${MAGNUM_SCENEGRAPH_LIBRARIES}` variable is probably not defined at all here and thus the AppVeyor build fails -- try `MagnumSceneGraph` instead